src/bundle/Resources/views/content/tab/translations/modal_add_translation.html.twig

```diff
@@ -6,7 +6,7 @@
         {{ form_start(form, {'action': path('ezplatform.translation.add')}) }}
         <div class="modal-content">
             <div class="modal-header">
-                <h5 class="modal-title">{{ 'tab.translations.add.title'|trans|desc('Create new translation') }}</h5>
+                <h2 class="modal-title">{{ 'tab.translations.add.title'|trans|desc('Create new translation') }}</h2>
```

Comment:
We need to think about semantics here. If you set it to `h2` then it semantically means it is a second important title in the HTML markup. If you would like to resize the title. Then maybe you should create a CSS selector that provides a desired font size definition.